demos/shortify/setup.py

```diff
@@ -22,7 +22,7 @@ def read_version():
     'trafaret',
     'aiohttp_jinja2',
     'pyyaml',
-    'aioredis==1.3.1'
```

Comment:
Could you replace this with `redis` as `aioredis` is no longer supported?
Related PRs, which might give some help:
https://github.com/aio-libs/aiohttp-session/pull/761
https://github.com/aio-libs/aiohttp-session/pull/701

Reply:
Sure. Thanks for the advice and links